lib/oli_web/views/layout_view.ex

```diff
@@ -35,6 +35,16 @@ defmodule OliWeb.LayoutView do
     )
   end
 
+  @doc """
+  Allows a delivery content template to specify any number of additional stylesheets, via URLs,
+  to be included in the head portion of the document.
+  """
+  def additional_stylesheets(assigns) do
+    Map.get(assigns, :additional_stylesheets, [])
+    |> Enum.map(fn url -> "\n<link rel=\"stylesheet\" href=\"#{url}\">" end)
```

Comment:
This could be a risk, if an author injects a dangerous html string in for url. Is this url protected from user input? To be safe, we could surround with escape_xml we use in rendering